core/build.go

```diff
@@ -35,7 +37,7 @@ func RestoreWorkingDirectory(workdir string, containerfile *os.File) error {
 	return nil
 }
 
-// BuildRecipe builds a Containerfile from a recipe path
+// Load and build the recipe
```

Comment:
```suggestion
// Load and build a Containerfile out of the given recipe
```

Reply:
I'd prefer `// Load and build a Containerfile from the specified recipe`

Reply:
yeah that would work too